src/lyapunovs.jl

```diff
@@ -254,7 +254,7 @@ function lyapunov(ds::ContinuousDynamicalSystem, T = 10000.0; Ttr = 0.0,
   st1 = ds.state
   integ1 = ODEIntegrator(ds, T; diff_eq_kwargs=diff_eq_kwargs)
   integ1.opts.advance_to_tstop=true
-  ds.state = st1 .+ displacement(st1, d0)
+  ds.state = displacement.(st1, d0)
```

Comment:
I think the dot should be at the equals, i.e. `.= displacement`

Now, the function itself definitely doesn't need a dot. It doesn't operate element-wise on an array, but takes an array and produces a new one.

So `ds.state .= displacement(st1, d0)`

Reply:
I mean, in many cases the user will specify a function that would not make sense to be broadcasted. This sacrifices some speed but gives flexibility.

Plus this part of the code is not the bottleneck so speed wouldn't matter here.